### PR TITLE
Fix coding style of cython files

### PR DESCRIPTION
**Pull request by okuta**

### CODE REVIEW

**Comment by unnonouno** [comment]
Do you have an idea to keep cython code clean?

**Comment by okuta** [comment]
I have no idea...

**Comment by unnonouno** [comment]
If so, it is difficult to keep the code clean, isn't it? I can't confirm if your fix is enough.

**Comment by okuta** [comment]
I found method to check cython code.
``find cupy/ \( -name \*.px\* -or -name \*.pyx \) -exec flake8 {} --ignore E901 \;``

**Comment by okuta** [comment]
@unnonouno I fixed. Please review this.

File: .travis.yml [hunk 1]
```diff
@@ -55,6 +55,7 @@ install:
 
 script:
   - flake8
+  - find cupy/ \( -name \*.px\* -or -name \*.pyx \) -exec flake8 {} --ignore E901,E225,E226,E227 \;
```
Comment (unnonouno) [inline .travis.yml]: ```flake8 --filename='*.pyx,*.px*' --ignore E901,E225,E226,E227``` is enough

Comment (okuta) [inline .travis.yml]: Thank you!
I will fix.

**Comment by unnonouno** [comment]
I'll merge this PR. It causes many conflicts in some PRs containing fixes in cython codes.

**Pull request merged**